### ISSUE TITLE
test aria-roledescription attribute

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#58]
This role would be very valuable, and could potentially solve problems such as
https://github.com/phetsims/balloons-and-static-electricity/issues/289

Here is some documentation from ets
http://www.ets-research.org/ia11ylab/aria11spec/

> The new aria-roledescription property provides content authors a mechanism to over-ride the default role name announced by screen readers with one that is potentially more meaningful to the task or context of use.

That article suggests VoiceOver supports this role.  We should test JAWS and NVDA as well.

**Comment by jessegreenberg** [comment]
aria-roledescription is not supported by NVDA, tested with latest version on 9/11/2017, NVDA reads the role and ignores roledescription.

**Comment by jessegreenberg** [comment]
I think we have all  the testing we need.  roledescription works with JAWS and VO, doesn't work with NVDA, falls back to default role name on that platform.

**Comment by jessegreenberg** [comment]
Update:  aria-roledescription works in IE11 + JAWS, but not FF + JAWS. Likely a browser issue then.
